core/state/statedb.go

```diff
@@ -30,6 +30,8 @@ import (
 	"github.com/ethereum/go-ethereum/trie"
 
 	"github.com/harmony-one/harmony/core/types"
+	"github.com/harmony-one/harmony/staking"
```

Comment:
duplicate of next line

Reply:
I added a new file under harmony/staking folder